docs/orchestrating-elastic-stack-applications/elasticsearch/readiness.asciidoc

```diff
@@ -30,7 +30,7 @@ spec:
                 command:
                 - bash
                 - -c
-                - /mnt/elastic-internal/scripts/readiness-probe-script.sh
+                - /mnt/elastic-internal/scripts/readiness-port-script.sh
```

Comment:
This is not correct pre 8.2.0 `probe` is the correct path.

Reply:
ah I got your point. I meant to point out that, in latest 2.14 version, the readiness probe changed from probe to port of it's name. I think I should put this somewhere but didn't figure out how to make it. I will rethink it and thanks for your comments.